skl2onnx/_supported_operators.py

```diff
@@ -168,8 +185,9 @@
     BernoulliNB, ComplementNB, GaussianNB, MultinomialNB,
     KNeighborsClassifier,
     CalibratedClassifierCV, OneVsRestClassifier, VotingClassifier,
-    AdaBoostClassifier, MLPClassifier, LinearDiscriminantAnalysis
-]
+    AdaBoostClassifier, MLPClassifier, LinearDiscriminantAnalysis,
+    HistGradientBoostingClassifier
```

Comment:
Comma at the end would help avoid modifying the last line when adding new elements.